.github/mergify.yml

```
@@ -3,6 +3,7 @@ shared:
   # Automated pull requests from bot users
   is_a_bot: &is_a_bot
     - or:
+      - "author=cloudpossebot"
```

YAMLlint (1.35.1)

[warning] wrong indentation: expected 8 but found 6

(indentation)